vck-openid/src/commonTest/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopInteropTest.kt

```diff
@@ -246,7 +246,7 @@ class OidcSiopInteropTest : FreeSpec({
         val parsed = jsonSerializer.decodeFromString<AuthenticationRequestParameters>(input)
         parsed.shouldNotBeNull()
 
-        parsed.responseUrl shouldBe "https://verifier-backend.eudiw.dev/wallet/direct_post"
+        parsed.responseUri shouldBe "https://verifier-backend.eudiw.dev/wallet/direct_post"
```

Comment:
Please rename `responseUri` to `responseUrl`, so that we don't introduce a breaking change